### PR TITLE
Use `zeebe-node@8.3.0-alpha4` to stub `tenantId`

### LINKED ISSUE #3772: Add stub support for tenantId
### What should we do?

Add the alpha package of zeebe-node to test the tenantId stub. 

### Why should we do it?

https://github.com/camunda/camunda-modeler/issues/3716

### PR DESCRIPTION
**Pull request by jwulf**
Closes #3772. 

Adds the alpha package of zeebe-node, and modifies the code to use the new APIs that support tenantId.

### CODE REVIEW

File: app/lib/zeebe-api/zeebe-api.js [hunk 1]
```diff
@@ -143,9 +143,11 @@ class ZeebeAPI {
     const client = await this._getZeebeClient(endpoint);
 
     try {
-      const response = await client.deployProcess({
-        definition: contents,
-        name: this._prepareDeploymentName(name, filePath, diagramType)
+      const response = await client.deployResource({
```
Comment (jwulf) [inline app/lib/zeebe-api/zeebe-api.js:146]: I'm actually not sure about this. Is this the code path to deploy only processes, or can you deploy DMN from here?

If so, the code would need to be different. See here: https://github.com/camunda-community-hub/zeebe-client-node-js/blob/multi-tenant-start/src/zb/ZBClient.ts#L728

Comment (nikku) [reply app/lib/zeebe-api/zeebe-api.js:146]: Good find! I'll checkout if this is actually an issue. We do allow deployment of DMN, too :see_no_evil:. But moving to the resource deployment API is an improvement one way or the other :tada:

**Comment by nikku** [comment]
Updated tests to use `deployResource` API.

**Comment by nikku** [comment]
I can verify that this still works, also when upgrading to the `deployResource` API. Thanks.

**Pull request merged**